### DISCUSSION TITLE
v1.1.7

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#301] [Announcements/公告]
<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.1.7'>v1.1.7</a>.</em>

**Comment by lowbibi** [comment]
大佬，建议蚂蚁庄园增加一个雇佣小鸡功能，蚂蚁新村增加一个邀请摆摊功能。

**Comment by heavenbeike** [comment]
神奇物种还有个任务是赠送一张物种卡就能多抽取一次，可以执行赠送指定的人或群(比如几个人拉一个群仅赠送不领取)，就能每天多抽取一次了

**Comment by xiaolanchong123** [comment]
只要能进来，不管什么功能，感谢大佬默默付出，赞一个

**Comment by xiaoqingshou** [comment]
感谢大佬的辛勤付出

**Comment by ZXW1991** [comment]
因为有大佬的辛苦付出，我们才能更加的愉快，辛苦辛苦！！！

**Comment by Lincope** [comment]
能不能默认关掉捐鸡蛋呀我攒的鸡蛋全给捐没了

00:33:29 捐赠活动❤️[困境儿童救助与保护]#累计捐赠250次
